neqo-transport/src/packet/retry.rs

```diff
@@ -10,32 +10,33 @@ use crate::version::Version;
 use crate::{Error, Res};
 
 use neqo_common::qerror;
-use neqo_crypto::{hkdf, Aead, TLS_AES_128_GCM_SHA256, TLS_VERSION_1_3};
+use neqo_crypto::{create_aead, hkdf, Aead, TLS_AES_128_GCM_SHA256, TLS_VERSION_1_3};
 
 use std::cell::RefCell;
 
 /// The AEAD used for Retry is fixed, so use thread local storage.
-fn make_aead(version: Version) -> Aead {
+fn make_aead(version: Version) -> Box<dyn Aead> {
     #[cfg(debug_assertions)]
     ::neqo_crypto::assert_initialized();
 
     let secret = hkdf::import_key(TLS_VERSION_1_3, version.retry_secret()).unwrap();
-    Aead::new(
+    create_aead(
+        false,
         TLS_VERSION_1_3,
         TLS_AES_128_GCM_SHA256,
         &secret,
         version.label_prefix(),
     )
     .unwrap()
 }
-thread_local!(static RETRY_AEAD_29: RefCell<Aead> = RefCell::new(make_aead(Version::Draft29)));
-thread_local!(static RETRY_AEAD_V1: RefCell<Aead> = RefCell::new(make_aead(Version::Version1)));
-thread_local!(static RETRY_AEAD_V2: RefCell<Aead> = RefCell::new(make_aead(Version::Version2)));
+thread_local!(static RETRY_AEAD_29: RefCell<Box<dyn Aead>> = RefCell::new(make_aead(Version::Draft29)));
+thread_local!(static RETRY_AEAD_V1: RefCell<Box<dyn Aead>> = RefCell::new(make_aead(Version::Version1)));
+thread_local!(static RETRY_AEAD_V2: RefCell<Box<dyn Aead>> = RefCell::new(make_aead(Version::Version2)));
```

Comment:
Even with this version of the code, you aren't providing a way to switch fuzzing on here, so you could always use the real AEAD, though I probably wouldn't bother.

Reply:
According to #1355, this will be tackled in another patch, though I am not sure what do do. Anyway, I'll skip this for now.